.github/workflows/rbc_test.yml

```diff
@@ -101,21 +101,21 @@ jobs:
         os: [ubuntu-latest]
         python-version: ['3.9', '3.8', '3.7']
         numba-version: ['0.55', '0.54']
-        heavydb-version: ['5.10', '5.9', '5.8', '5.7']
+        heavydb-version: ['6.0', '5.10', '5.9']
         heavydb-from: [conda]
-        include:
-          - os: ubuntu-latest
-            python-version: '3.10'
-            numba-version: '0.55'
-            heavydb-version: dev
-            docker-image: omnisci/core-os-cpu-dev:latest
-            heavydb-from: docker
-          - os: ubuntu-latest
-            python-version: '3.9'
-            numba-version: '0.54'
-            heavydb-version: dev
-            docker-image: omnisci/core-os-cpu-dev:latest
-            heavydb-from: docker
+        # include:
+        #   - os: ubuntu-latest
+        #     python-version: '3.10'
+        #     numba-version: '0.55'
+        #     heavydb-version: dev
+        #     docker-image: heavyai/core-os-cpu-dev:latest
+        #     heavydb-from: docker
+        #   - os: ubuntu-latest
+        #     python-version: '3.9'
+        #     numba-version: '0.54'
+        #     heavydb-version: dev
+        #     docker-image: heavyai/core-os-cpu-dev:latest
+        #     heavydb-from: docker
```

Comment:
What is the plan with this commented-out section? Is `heavyai/core-os-cpu-dev:latest` available?

Reply:
It is available now

Reply:
The `-dev` images are not available.